### ISSUE TITLE
Confused: Empty message or reading notification?

### ISSUE THREAD

**Issue by androidgod** [redsolution/xabber-android#255]
![](http://abload.de/img/empty_message_xabber46ap1.png)

**Comment by chaoscommander** [comment]
I believe this started happening when pidgin-otr-4.0.0 came out, probably it's a protocol change. Pidgin to Pidgin doesn't have the problem. This tells me OTR has been outdated on Xabber since 2012. *cough*

**Comment by White-Tiger** [comment]
@chaoscommander you're right and it's likely about OTRv3 because this issue also occurs if the other client is Miranda NG which also supports OTRv3

To be honest, porting from OTRv2 to OTRv3 is a little bit of extra work... especially if one wants to support all OTRv3 features such as multiple instance tags.
